stories/Content Presentation/Adult Bmi/Figure.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { mockedBmiClassificationBounds } from '@tests/__helpers__/mocks/bmi-results';
+import { BmiChartFigure } from '@components/bmi/bmi-figure-chart';
+
+const meta: Meta<typeof BmiChartFigure> = {
+  title: 'Content Presentation/Adult Bmi/Figure',
+  component: BmiChartFigure,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof BmiChartFigure>;
+/**
+ *
+ * This component can be found in the `nhsuk-tools-chart-components` repository <a href="https://github.com/NHSDigital/nhsuk-tools-chart-components/tree/main/src/components/bmi" target="_blank" rel="noopener noreferrer">here</a>.
+ *
+ * ## Usage
+ *
+ * ```jsx
+ *  <BmiChartFigure
+ *     ariaLabel="Your body mass index result is shown on a chart here. A full description can be found in the text below it."
+ *     classificationBounds={classificationBounds}
+ *     bmi={19.4}
+ *     legendMarkerText="Your BMI"
+ *     legendKeys={{
+ *       underweight: 'Underweight',
+ *       healthy: 'Healthy',
+ *       overweight: 'Overweight',
+ *       obese: 'Obese',
+ *     }}
+ *   />
+ * ```
+ *
+ * ### Properties
+ * ## Classification bounds
```

Comment:
should this be #### rather than ## ? do we need to add the rest of the properties/parameters in here too?

Reply:
headings fixed and all props have been included in the storybook comments